snprc_ehr/resources/source_queries/therapy/create_v_delete_therapy_formulary_and_lookups.sql

```diff
@@ -0,0 +1,59 @@
+CREATE VIEW labkey_etl.v_delete_therapy_formulary_and_lookups AS
```

Comment:
Hi Terry. Can i please know the purpose of this delete therapy etl? why do we use this and the functionality around this ?